### ISSUE TITLE
Add support for dbt version 1.4

### ISSUE THREAD

**Issue by siljamardla** [slidoapp/dbt-coverage#59]
I am using dbt version 1.4.5:
```
dbt --version
Core:
  - installed: 1.4.5
  - latest:    1.4.5 - Up to date!
```
when I run 
`dbt-coverage compute doc`
I get the following error about incompatibility
```
WARNING:root:Unsupported manifest.json version https://schemas.getdbt.com/dbt/manifest/v8.json, 
unexpected behavior can occur. 
Supported versions: [
'https://schemas.getdbt.com/dbt/manifest/v4.json', 
'https://schemas.getdbt.com/dbt/manifest/v5.json', 
'https://schemas.getdbt.com/dbt/manifest/v6.json', 
'https://schemas.getdbt.com/dbt/manifest/v7.json'
]. 
See https://github.com/slidoapp/dbt-coverage/tree/main#supported-dbt-versions for more details.
```
and the resulting stats are always 0/xxx columns documented.

**Comment by sweco** [comment]
Hey @siljamardla, sorry for a late reply. We were always testing our releases against the [`jaffle_shop` project](https://github.com/dbt-labs/jaffle_shop) and it worked. I just tested it with `dbt-core==1.4.6` and it works just fine.

However, we had problems with aliases https://github.com/slidoapp/dbt-coverage/issues/30. I just released [`dbt-coverage==0.3.3`](https://github.com/slidoapp/dbt-coverage/releases/tag/v0.3.3) that should fix this. Are you using aliases by any chance? If yes, could you please check whether it works?

**Comment by siljamardla** [comment]
Interesting. I can't remember doing any upgrades, but here's what I get now:

```
WARNING:root:Unsupported manifest.json version https://schemas.getdbt.com/dbt/manifest/v8.json, unexpected behavior can occur. Supported versions: ['https://schemas.getdbt.com/dbt/manifest/v4.json', 'https://schemas.getdbt.com/dbt/manifest/v5.json', 'https://schemas.getdbt.com/dbt/manifest/v6.json', 'https://schemas.getdbt.com/dbt/manifest/v7.json']. See https://github.com/slidoapp/dbt-coverage/tree/main#supported-dbt-versions for more details.
```
As opposed to last time, it does actually produce some stats.

**Comment by sweco** [comment]
@siljamardla, hmmm... That's intriguing indeed. Are you using `dbt-coverage==0.3.3` or a different version?

Also, I can see that the issue had 3 upvotes. @StepienTomasz, @vvvito, @wmorin, did you also experience the issue with 0 coverage or you simply upvoted because of the warning?

**Comment by sweco** [comment]
@siljamardla, we've just released [version 0.3.4](https://github.com/slidoapp/dbt-coverage/releases/tag/v0.3.4) that removes the warning. Please, do let me know if you the zero coverage happens again.
